thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/ThirdEyeDashboardApplication.java

```diff
@@ -160,14 +155,12 @@ public void run(ThirdEyeDashboardConfiguration config, Environment env)
 
     AnomalyFunctionFactory anomalyFunctionFactory = new AnomalyFunctionFactory(config.getFunctionConfigPath());
     AlertFilterFactory alertFilterFactory = new AlertFilterFactory(config.getAlertFilterConfigPath());
```

Comment:
AlertFilterFactory is not used.
All the classes under org.apache.pinot.thirdeye.detector.email.filter are not used.

Reply:
Same as above.